src/handlers/records-write.ts

```diff
@@ -163,7 +163,9 @@ export class RecordsWriteHandler implements MethodHandler {
     }
 
     const messageReply = {
-      status: { code: 202, detail: 'Accepted' }
+      status: (newMessageIsInitialWrite && dataStream === undefined) ?
```

Comment:
Nit: Would be good to add some context in a comment here. I know we mention a bit about writing without data above, but it might be good to add a small comment explaining why we return `No Content`.

Reply:
Added a comment.